File: C#/Урок 040/Задача 04.cs
class Program
{
    static void Main()
    {
        System.Console.Write("Enter a positive number: ");
        int userNumber = System.Convert.ToInt32(System.Console.ReadLine());

        int sum = 0;
        for (int number = 1; number <= userNumber; number++)
            sum += number;

        System.Console.WriteLine("Sum = " + sum);
    }
}

File: C++/Урок 034 - for/Задача 04.cpp
#include <iostream>

int main()
{
    // Input number:
    int userNumber;
    do
    {
        std::cout << "Enter a positive number: ";
        std::cin >> userNumber;
    } while (userNumber <= 0);

    int sum = 0;
    for (int number = 1; number <= userNumber; number++)
    {
        sum += number;
    }

    std::cout << "Sum = " << sum << std::endl;

    return 0;
}